### ISSUE TITLE
How use the next_page in query

### ISSUE THREAD

**Issue by Damzou** [prismicio/javascript-kit#136]
Hello,

We are developping an application with prismic.io's javascript-kit and we need help to understand how to use the next_page attribute when we ave more of 100 entries in a document.
We tried :
 - to put the url (next_page) in query : .query(response.next_page).submit(...) // returns error 400 bad request
 - to put it in Api.form : Api.form(response.next_page) // returns null

There is no informations or examples in the documentation.
Any help would be nice.

**Comment by dohzya** [comment]
The `next_page` attribute is the exact URL of the next page of the request (only misses the access token), so it is not made to be used in a form.

A helper is on its way to get the next page more easily, but in the meantime, you can use the `Api.request` method. Note that you will have to add the access token in the URL, and this method currently only support the callback style (it doesn't return a Promise yet).

```js
var url = page1.next_page + (this.accessToken ? '&access_token=' + Api.accessToken : '');
Api.request(url, function (err, page2) { ... });
```

**Comment by arnaudlewis** [comment]
@Damzou It's fixed since 3.4.0.
If you wanna use it : https://github.com/prismicio/javascript-kit/blob/master/lib/api.js#L514
Just call api.getNextPage
